configure

```diff
@@ -5999,17 +5999,31 @@ append_var SONIVOX_LIBS "-lsonivox"
 if test "$_sonivox" = auto; then
 	_sonivox=no
 	cat > $TMPC << EOF
+#include <stdio.h>
 #include <sonivox/eas.h>
-int main(void) { /* delete_fluid_settings(new_fluid_settings()); */ return 0; }
+
+EAS_DATA_HANDLE mEASDataHandle = NULL;
+
+int main(void) {
+	EAS_Init(&mEASDataHandle);
+	return 0;
+}
 EOF
 	cc_check_no_clean $SONIVOX_CFLAGS $SONIVOX_LIBS && _sonivox=yes
 	if test "$_sonivox" != yes && test "$_pkg_config" = "yes" && $_pkgconfig --exists sonivox; then
 		SONIVOX_LIBS="`$_pkgconfig --static --libs sonivox`"
 		cc_check_no_clean $SONIVOX_CFLAGS $SONIVOX_LIBS && _sonivox=yes
 	fi
-	if test "$_sonivox" != yes && test "$_pkg_config" = "yes" && $_pkgconfig --exists sonivox-static; then
-		SONIVOX_LIBS="`$_pkgconfig --static --libs sonivox-static`"
-		cc_check_no_clean $SONIVOX_CFLAGS $SONIVOX_LIBS && _sonivox=yes
+	if test "$_sonivox" != yes ; then
```

Comment:
Shouldn't this be duplicated for sonivox case above?
From upstream pkg-config, they only need libm which we already add: this should be quite safe.

Reply:
Lines [5998](https://github.com/scummvm/scummvm/blob/63c600d9d6ac15275b2e33f306c84ea1bbb5bb24/configure#L5998) and 6012 already cover that case, I believe?

Reply:
What I meant is that you don't check for `-lsonivox` without pkg-config

Reply:
Yes, I understood that, but read line 6005/6012: it was _already_ doing a -lsonivox check *before* the pkg-config checks were done.

So I didn't add it because it was already covered.